gnovm/cmd/gno/run_test.go

```diff
@@ -69,7 +69,7 @@ func TestRunApp(t *testing.T) {
 		},
 		{
 			args:                []string{"run", "-debug", "../../tests/integ/debugger/sample.gno"},
-			stdoutShouldContain: "Welcome to the Gnovm debugger",
+			stdoutShouldContain: "in main\nhello 3\n!zero\n2\nbye 4",
```

Comment:
Why did this change?

Reply:
Because `Welcome to the Gnovm debugger` isn't what is returned from that test.  I made what maybe an erroneous assumption that this was inadvertently broken earlier/elsewhere, but it was causing my test runs to fail.

Reply:
:bulb: 

I figured it out.....

Well, partly.  It was a bad assumption on my part.  Solving it.

Reply:
This should be resolved, now, pending the completion of the CI.